website/_guides/installation.md

```diff
@@ -0,0 +1,222 @@
+
+```bash
+$ npm install --global typedoc
+```
+
+Like the TypeScript compiler, TypeDoc comes with a binary that can be called from anywhere if you
+install TypeDoc as a global module. The name of the executable is ``typedoc``. To verify everything
+is setup correctly, you can run TypeDoc with the ``--version`` argurment. It should output something like the following:
+
+```bash
+$ typedoc --version
+
+TypeDoc 0.13.0
+Using TypeScript 3.1.1 from /Users/aciccarello/Documents/code/typedoc/typedoc/node_modules/typescript/lib
+```
+
+
+### Local installation
+
+If you want to use TypeDoc as a node module within your project or as an npm script, you can
+install it as a local module. This is recommended for uses such as integrating it with your build process.
+By saving to the project `package.json` file,
+anyone who runs `npm install` on the project will have typedoc installed.
+
+```bash
+$ npm install --save-dev typedoc
+```
+
+## Command line interface
+The CLI can be used both from your terminal or from npm scripts. All argurments that are not passed
+with flags are considered input files or directories. Use either the ``--out`` or ``--json``
+arguments to define the format and destination of your documentation.
+
+```bash
+typedoc --out docs /src
+```
+
+### JSON Configuration
+Instead of passing all arguments via the command line, the CLI also supports reading TypeDoc configuration from json files.
+
+### typedoc.json
+When running typedoc from the CLI, you can define any option except the entry files in a json file named `typedoc.json`.
+
+```json
+{
+    "mode": "file",
+    "out": "docs"
+}
+```
+
+### tsconfig.json
+TypeDoc options can be defined withing an existing `tsconfig.json` file. Use a `typedocOptions` section to define
+options as a json model.
+```json
+{
+    "compilerOptions": {
+      "normalTypeScriptOptions": "here"
+    },
+    "typedocOptions": {
+        "mode": "modules",
+        "out": "docs"
+    }
+}
+```
+
+## Node module
+If you would like dynamic configuration or would like to run typedoc without using the CLI, import
+the node module and build the documentation yourself.
+```javascript
+const TypeDoc = require('typedoc');
+
+const app = new TypeDoc.Application({
+    mode:   'Modules',
+    logger: 'none',
+    target: 'ES5',
+    module: 'CommonJS',
+    experimentalDecorators: true
+});
+
+const project = app.convert(app.expandInputFiles(['src']));
+
+if (project) { // Project may not have converted correctly
+    const outputDir = 'docs';
+
+    // Rendered docs
+    app.generateDocs(project, outputDir);
+    // Alternatively generate JSON output
+    app.generateJson(project, outputDir + '/documentation.json');
+}
+```
+
+## Third-Party Tools
+
+Some great folks out there have created plugins so you can run TypeDoc with your favorite task runner.
+Plugins are available for Grunt, Gulp, and Webpack. All of them can be installed using ``npm``.
+
+
+### Grunt
```

Comment:
I was under the impression that no grunt plugin was required for typedoc, why else do we have [tasks/typedoc.js](https://github.com/TypeStrong/typedoc/blob/master/tasks/typedoc.js)?

I don't use grunt anywhere else, but the linked plugin hasn't been updated in years.

Reply:
That's been in the docs for years. I assume the grunt plugin still works and I think it'd be worth leaving it in there if it does.